spec/paperclip/media_type_spoof_detector_spec.rb

```diff
@@ -43,4 +43,18 @@
       Paperclip.options[:content_type_mappings] = {}
     end
   end
+
+  it 'does allow array as :content_type_mappings' do
+    begin
+      Paperclip.options[:content_type_mappings] = {
+        html: ['binary', 'text/html']
+      }
+      file = File.open(fixture_file('empty.html'))
+      spoofed = Paperclip::MediaTypeSpoofDetector
+                .using(file, 'empty.html').spoofed?
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.